tests/integrations/test_db_integrations_sql.py

```diff
@@ -343,16 +404,52 @@ def test_db_user(self):
         super().test_db_user()
 
 
-############################################################
-# Mssql: Tracer and Agent validations
-############################################################
 @scenarios.integrations
-class Test_Tracer_Mssql_db_integration(_BaseTracerIntegrationsSqlTestClass):
+class Test_Tracer_Mysql_db_integration(_BaseTracerIntegrationsSqlTestClass, _Base_Mysql_db_integration):
+    """ Overwrite or add specific validation methods for mysql on tracer interface """
+
+    pass
+
+
+@scenarios.integrations
+class Test_Agent_Mysql_db_integration(_BaseAgentIntegrationsSqlTestClass, _Base_Mysql_db_integration):
+    """ Overwrite or add specific validation methods for mysql on agent interface """
+
+    pass
+
+
+@scenarios.otel_integrations
+class Test_Agent_Mysql_db_otel_integration(_BaseOtelAgentIntegrationsSqlTestClass, _Base_Mysql_db_integration):
+    """ Overwrite or add specific validation methods for mysql on agent interface (app instrumented by open telemetry) """
+
+    def test_obfuscate_query(self):
+        """ All queries come out obfuscated from agent """
+        for db_operation, request in self.requests[self.db_service].items():
+            span = self._get_sql_span_for_request(request)
+            if db_operation in ["update", "delete", "procedure"]:
+                assert (
+                    span["meta"]["db.statement"].count("?") == 2
+                ), f"The query is not properly obfuscated for operation {db_operation}"
+            elif db_operation is "select_error":
+                continue
```

Comment:
you request the span, without testing anything, if it's on purpose, maybe a small comment explaining why can help

Reply:
It was a mistake! Solved! Thanks!